packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec

```diff
@@ -98,6 +97,9 @@ install -m 0644 .%{gem_instdir}/config/%{plugin_name}.yml \
 %{gem_instdir}/test
 
 %changelog
+* Thu Feb 13 2019 Zach Huntington-Meath - 0.21-0.3.pre.master
```

Comment:
Linter doesn't like you using last year in the date ;)

Reply:
Linter is so picky